youtube_dl/extractor/bandcamp.py

```diff
@@ -415,3 +408,52 @@ def _real_extract(self, url):
             'episode_id': compat_str(video_id),
             'formats': formats
         }
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:user'
+    _VALID_URL = r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com/?$'
+
+    _TESTS = [{
+        'url': 'https://adrianvonziegler.bandcamp.com',
+        'info_dict': {
+            'id': 'adrianvonziegler',
+            'title': 'Albums of adrianvonziegler',
+        },
+        'playlist_mincount': 20,
+    }, {
+        'url': 'http://dotscale.bandcamp.com',
+        'info_dict': {
+            'id': 'dotscale',
+            'title': 'Albums of dotscale',
+        },
+        'playlist_count': 1,
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        uploader = mobj.group('subdomain')
+
+        webpage = self._download_webpage(url, uploader)
+
+        album_elements = re.findall(r'<a href="/album/(.+)">', webpage)
```

Comment:
Capture with /album. Capture non greedy.

Reply:
I can't capture with just /album as there are other instances that match which I don't want.

Nevertheless the non greedy capture is done in 45f47dbaa0a057dbd1daef4aba517dc947ce6853

Reply:
This does not make any difference since you prepend album anyway.

Reply:
Sorry @dstftw I wasn't understanding what you meant. Now is fixed e8f95092a